docker-compose.yml

```diff
@@ -8,15 +8,14 @@ services:
       - "80:80"
     volumes:
       - ~/www:/var/www/html
+      - ~/docker/proximis/nginx:/etc/nginx/conf.d
```

Comment:
It's not needed if you keep the copy instruction in the Dockerfile.
https://github.com/EmakinaFR/docker-proximis/blob/proximis-2.0/nginx/Dockerfile#L5

Reply:
Fix